### DISCUSSION TITLE
[Networking] Azure function - Consumption plan

### DISCUSSION THREAD

**Discussion by jakubigla** [AzureCommunityPL/group#72] [Pomoc]
Czesc,

Pewnie podstawy, ALE: Czy jesli moje Azure functions, ktore wykorzystuja consumption plan (czyli nie maja dostepu do VNETu), beda chcialy sie polaczyc z managed postgresem albo storage account, to czy traffic opuszcza Azure backbone network czy nie? 
Mam same managed servicy, zero IaaS.

**Comment by kamil-mrzyglod** [comment]
Ale rozumiesz przez "managed" że masz po prostu standardową instancję PostgreSQL / Storage Account i łączysz się z nimi po publicznych adresach?

**Comment by piotrgwiazda** [comment]
W Postgresie możesz wyłączyć dostęp publiczny i kliknąć Allow access for
Azure services. Co prawda to sieciowo ogranicza dostęp do usług z Azure'a,
ale nie tylko Twoich. Opuszczanie sieci MS w takim modelu byłoby bez sensu,
ale czy jest na to 100% gwarancji to nie wiem. Nie wiem też jaki problem to
rozwiązuje. Egress fees nie zapłacisz na pewno jeśli to ten sam region.

niedz., 20 wrz 2020, 10:51 użytkownik Kamil Mrzygłód <
notifications@github.com> napisał:

**Comment by smereczynski** [comment]
Do przedmówców: chyba nie złapaliście clue pytania. W Azure ruch jest realizowany według zasady zimnego ziemniaka (cold potato). To znaczy, że pakiety zostają w backbone MS tak długo, jak tylko się da.

**Reply by kamil-mrzyglod** [reply]
True, nawet nie wiedziałem, że to nazywa sie "zimny ziemniak" a faktycznie jest w dokumentacji ;) https://docs.microsoft.com/en-us/azure/virtual-network/routing-preference-overview

**Comment by piotrgwiazda** [comment]
Jest teraz opcja do wyboru
[image: image.png]

niedz., 20 wrz 2020 o 20:40 michal_s <notifications@github.com> napisał(a):
